sonic-ycabled/ycable/ycable.py

```diff
@@ -54,6 +54,8 @@
 NOT_IMPLEMENTED_ERROR = 3
 SFP_SYSTEM_ERROR = 4
 
+MUX_CABLE_STATIC_INFO_TABLE = "MUX_CABLE_STATIC_INFO"
+MUX_CABLE_INFO_TABLE = "MUX_CABLE_INFO"
```

Comment:
Can we put table names in schema.h and replace the hardcoded ones in the future?

Reply:
will open a seperate PR for that, since chnage needs to come from swsscommon